raw_description/urdf/base_long/base.urdf.xacro

```diff
@@ -117,12 +117,12 @@
 
 
     <!-- base laser front -->
-    <xacro:sick_s300_laser_v0 name="${name}_laser_front" parent="${name}" ros_topic="scan_front_raw" update_rate="10" min_angle="-2.3562" max_angle="2.3562" >
+    <xacro:sick_s300_laser_v0 name="${name}_laser_front" parent="${name}" ros_topic="laser_front/scan_raw" update_rate="10" min_angle="-2.3562" max_angle="2.3562" >
```

Comment:
same here:
if the topic name is appended to the sensor name, we could get rid of the ros_topic param and just set it to `scan_raw`

These kind of things need to be in sync with the according hardware drivers, e.g. https://github.com/ipa320/cob_driver/blob/indigo_dev/cob_sick_s300/ros/src/cob_sick_s300.cpp#L202